redbot/core/dev_commands.py

```diff
@@ -336,17 +336,33 @@ async def mock(self, ctx, user: discord.Member, *, command):
 
     @commands.command(name="mockmsg")
     @checks.is_owner()
-    async def mock_msg(self, ctx, user: discord.Member, *, content: str):
+    async def mock_msg(self, ctx, user: Union[discord.Member, discord.User, int], *, content: str):
         """Dispatch a message event as if it were sent by a different user.
 
         Only reads the raw content of the message. Attachments, embeds etc. are
         ignored.
         """
+        if isinstance(user, int):
+            try:
+                user = await self.bot.fetch_user(user)
+            except discord.NotFound:
+                raise Exception("User not found") #change this to some other Exception
```

Comment:
Don't raise bare exceptions inside a command

Reply:
uh i made a comment for you to change it becuase im not sure which exceptions you ppl use.